.circleci/config.yml

```diff
@@ -0,0 +1,57 @@
+version: 2
+
+# https://circleci.com/blog/circleci-hacks-reuse-yaml-in-your-circleci-config-with-yaml/
+defaults: &defaults
+  docker:
+    - image: circleci/golang:1.11.1-stretch
+  working_directory: /go/src/github.com/weaveworks/kubediff
+  environment:
+    GOPATH: /go
+
+workflows:
+  version: 2
+  build-test-deploy:
+    jobs:
+      - build-test
+      - deploy:
+          requires:
+            - build-test
+          filters:
+            branches:
+              only: master
+
+jobs:
+  build-test:
+    <<: *defaults
+    steps:
+    - checkout
+    - run:
+        name: Dependencies
+        command: |
+          pip install virtualenv
```

Comment:
```suggestion
          apt install python-pip
          pip install virtualenv
```

Reply:
Haven't tested it, but it looks like `pip` is missing (https://circleci.com/gh/weaveworks/kubediff/150) and we were using pip and python2 before: https://github.com/weaveworks/kubediff/blob/1501a2e89b8a8ac0fe3b21daecc2b6994be2703c/Dockerfile#L5

Reply:
It's working now; perhaps you caught it in one of the variants I tried.